### PR TITLE
tahmeed/CHM-72-ABS-NR1-Part-III

### PR DESCRIPTION
**Pull request by tahmeedshah**
### General description
ABS-NR1-Part-III copy text to translation file
ABS-NR1-Part-III questions structure

### Designs
_Link to the related design prototypes (if applicable)_

### Testing instructions
_Provide minimal instructions on how to test this PR._

* Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
* [ ]  Branch name / PR includes the related Jira ticket Id.
* [ ]  Tests to check core implementation / bug fix added.
* [ ]  All checks in Continuous Integration workflow pass.
* [ ]  Feature functionally tested by reviewer(s).
* [ ]  Code reviewed by reviewer(s).
* [ ]  Documentation updated (README, CHANGELOG...) (if required)

### CODE REVIEW

File: app/app-data/abs/report-analyzer/npNationalReport1.js [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+            "key": "Q007_a",
+            "section": "Part-III",
+            "number": "7.1",
+            "type": "option",
+            "title": nr1T.part_III_question7_1_title,
+            "multiple": false,
+            "mandatory": true,
+            "options": [
+               {
+                  "value": "60CD889F-4E08-450C-A67C-5F29A2B1E704",
```
Comment (blaisefonseca) [inline app/app-data/abs/report-analyzer/npNationalReport1.js:286]: can we use true and true.some like bch

File: app/app-data/abs/report-analyzer/npNationalReport1.js [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+         },
+         {
+            "key": "Q012_a",
+            "section": "Part-III",
+            "number": "12.1",
+            "type": "int",
+            "title": nr1T.part_III_question12_1_title,
+            "multiple": false,
+            "mandatory": true
+         },
```
Comment (blaisefonseca) [inline app/app-data/abs/report-analyzer/npNationalReport1.js:544]: image
is missing

File: app/app-data/abs/report-analyzer/npNationalReport1.js [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+            "section": "Part-III",
+            "number": "12.2",
+            "type": "option",
+            "title": nr1T.part_III_question12_2_title,
+            "multiple": false,
+            "mandatory": true,
+            "options": [
+               {
+                  "value": "true",
+                  "title": nr1T.yesInAllCases
```
Comment (blaisefonseca) [inline app/app-data/abs/report-analyzer/npNationalReport1.js:556]: ` "type": "lstring",
                  "caption": nr1T.pleaseExplainYourResponse` is missing for Yes

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
see comments

File: app/app-data/abs/report-analyzer/npNationalReport1.js (hunk 3 above)
Comment (tahmeedshah) [reply app/app-data/abs/report-analyzer/npNationalReport1.js:556]: image

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**